frontend/src/utils/giteaInternalApi.js

```diff
@@ -1,8 +1,14 @@
+/*
+* Committing multiple files to gitea is a two steps process:
+*  1. Upload the files to gitea which it turn returns a UUID for that file
+*  2. UUID returned from the previous step is used to make the actual commit request.
+*/
+
 /**
- *
- * @param repo
+ * Upload a file to gitea server, just upload it doesn't commit the files
```

Comment:
Huh? What's your plan here? I think you might over-complicating this? I think we should upload and commit multiple files in one request.